easybuild/toolchains/compiler/llvm.py

```diff
@@ -0,0 +1,191 @@
+##
+# Copyright 2013-2025 Ghent University
+#
+# This file is triple-licensed under GPLv2 (see below), MIT, and
+# BSD three-clause licenses.
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for Clang + Flang compiler toolchain.
+
+Authors:
+
+* Dmitri Gribenko (National Technical University of Ukraine "KPI")
+* Davide Grassano (CECAM EPFL)
+"""
+
+from easybuild.tools import LooseVersion
+import easybuild.tools.systemtools as systemtools
+from easybuild.tools.toolchain.compiler import Compiler, DEFAULT_OPT_LEVEL
+from easybuild.tools.toolchain.toolchain import SYSTEM_TOOLCHAIN_NAME
+
+TC_CONSTANT_LLVM = "LLVM"
+
+
+class LLVM(Compiler):
+    """Compiler toolchain with Clang and GFortran compilers."""
```

Comment:
"GFortran" was copy/pasted from somewhere else?
This one should be Clang + flang, no?

Reply:
Yeah i think they were leftovers when i restarted this files from the GCC ones thanks for catching these errors, fixed in [a9080ed](https://github.com/easybuilders/easybuild-framework/pull/4914/commits/a9080ed9bf31c69147dfe7c095497b741974a30e)